src/components/calculator.js

```diff
@@ -1,35 +1,52 @@
-        <div className="display_board">0</div>
-        <button type="button" className="btn">AC</button>
-        <button type="button" className="btn">+/-</button>
-        <button type="button" className="btn">%</button>
-        <button type="button" className="btn operator">÷</button>
-        <button type="button" className="btn">7</button>
-        <button type="button" className="btn">8</button>
-        <button type="button" className="btn">9</button>
-        <button type="button" className="btn operator">x</button>
-        <button type="button" className="btn">4</button>
-        <button type="button" className="btn">5</button>
-        <button type="button" className="btn">6</button>
-        <button type="button" className="btn operator">-</button>
-        <button type="button" className="btn">1</button>
-        <button type="button" className="btn">2</button>
-        <button type="button" className="btn">3</button>
-        <button type="button" className="btn operator">+</button>
-        <button type="button" className="btn zero">0</button>
-        <button type="button" className="btn">.</button>
-        <button type="button" className="btn operator">=</button>
+        <div className="display_board">
+          { total }
+          { operation }
+          { next }
+        </div>
+        <button type="button" onClick={this.handleClick} className="btn">AC</button>
+        <button type="button" onClick={this.handleClick} className="btn">+/-</button>
+        <button type="button" onClick={this.handleClick} className="btn">%</button>
+        <button type="button" onClick={this.handleClick} className="btn operator">÷</button>
+        <button type="button" onClick={this.handleClick} className="btn">7</button>
+        <button type="button" onClick={this.handleClick} className="btn">8</button>
+        <button type="button" onClick={this.handleClick} className="btn">9</button>
+        <button type="button" onClick={this.handleClick} className="btn operator">x</button>
+        <button type="button" onClick={this.handleClick} className="btn">4</button>
+        <button type="button" onClick={this.handleClick} className="btn">5</button>
+        <button type="button" onClick={this.handleClick} className="btn">6</button>
+        <button type="button" onClick={this.handleClick} className="btn operator">-</button>
+        <button type="button" onClick={this.handleClick} className="btn">1</button>
+        <button type="button" onClick={this.handleClick} className="btn">2</button>
+        <button type="button" onClick={this.handleClick} className="btn">3</button>
+        <button type="button" onClick={this.handleClick} className="btn operator">+</button>
+        <button type="button" onClick={this.handleClick} className="btn zero">0</button>
+        <button type="button" onClick={this.handleClick} className="btn">.</button>
+        <button type="button" onClick={this.handleClick} className="btn operator">=</button>
```

Comment:
- [ ] According to the project requirements, you are supposed to pass props to a child component of the `Calculator` component. I will like to advise you to create a component for displaying the calculations or a Button component for displaying the buttons and handling their `onClick` functions. You could also decide to make both children components of the Calculator component. 👍